.github/workflows/upgrade-python-requirements.yml

```diff
@@ -44,7 +49,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8"]
+        python-version: [${{ inputs.python_version }}]
```

Comment:
Isn't it odd to make a one-element matrix?  We could put the python version on line 68 instead, and have no matrix.